### ISSUE TITLE
[Bug] Additional Statamic fields being removed on product sync

### ISSUE THREAD

**Issue by sebszocinski** [jackabox/statamic-shopify#30]
I added a new field called `short_description` to my product blueprint and added some content to a product, then updated and synced from Shopify and noticed it wiped my `short_description`. It also removed the `updated_by` and `updated_at` and modified the `vendor` and `type` fields from strings to arrays. See below:

### Original Import:
```
---
product_id: 6622716625061
published_at: '2021-04-02 17:25:11'
title: 'Fill Up My Purchase Hole'
vendor: 'XX Shop'
type: Print
featured_image: Shopify/IMG_8024.jpg
gallery:
  - Shopify/IMG_8024.jpg
id: ba771be1-8f56-40a1-bdc2-d2faf65ece46
---
Giclee Gloss Print
```

### Edit in Statamic with new field:
```
---
product_id: 6622716625061
published_at: '2021-04-02 17:25:00'
title: 'Fill Up My Purchase Hole'
vendor:
  - 'XX Shop'
type:
  - Print
featured_image: Shopify/IMG_8024.jpg
gallery:
  - Shopify/IMG_8024.jpg
short_description: 'House paint on canvas. 2019'
updated_by: 97dff0d7-f47e-4f45-91e8-9942d2e03096
updated_at: 1617583665
id: ba771be1-8f56-40a1-bdc2-d2faf65ece46
---
<p>Giclee Gloss Print</p>
```

### After new sync from Shopify:
```
---
product_id: 6622716625061
published_at: '2021-04-02 17:25:11'
title: 'Fill Up My Purchase Hole'
vendor: 'XX Shop'
type: Print
featured_image: Shopify/IMG_8024.jpg
gallery:
  - Shopify/IMG_8024.jpg
id: ba771be1-8f56-40a1-bdc2-d2faf65ece46
---
Giclee Gloss Print
```

Edit: I am using version 1.2 but i'm also on Statamic 3.1.0-alpha.4

**Comment by klickreflex** [comment]
I'm having the same problem with some additional fields for SEO and OG tags and images, which I'd like to add to all my collections but can't use on products because they're being removed completely on every import.

**Comment by jackabox** [comment]
Fixed this in [1.3.0](https://github.com/jackabox/statamic-shopify/releases/tag/1.3.0). All additional data should hold now :)
